schemas/bagel_schema.json

```diff
@@ -21,26 +21,33 @@
         "has_mri_data": {
             "Description": "Whether or not participant had MRI data acquired in a given session.",
             "dtype": "bool",
-            "IsRequired": true,
+            "IsRequired": false,
             "Range": [true, false],
             "IsPrefixedColumn": false
         },
-        "DATATYPE_": {
-            "Description": "Whether or not participant session has specified BIDS datatype. Column suffix should correspond to a specific BIDS subdirectory, e.g., 'anat'.",
+        "HAS_DATATYPE__": {
+            "Description": "Whether or not participant session has specified raw BIDS datatype. Column suffix should correspond to a specific BIDS subdirectory. e.g., 'HAS_DATATYPE__anat'",
+            "dtype": "bool",
+            "IsRequired": false,
+            "Range": [true, false],
+            "IsPrefixedColumn": true
+        },
+        "HAS_CONTRAST__": {
```

Comment:
Is there a reason to name this `HAS_CONTRAST` vs `ACQUISITION`?

Reply:
Wouldn't acquisition be `grappa`/`adni`/etc.? What does BIDS call `T1w`/`T2w`/`FLAIR`? Looks like it's `suffix` but that can be confusing for the bagel

Reply:
Yes, I was just reading BIDS spec again for this :) 
`ACQ` and `CONTRAST` (e.g. contrast agent like Gad) both have more specific meaning in BIDS, so I am leaning towards calling it `suffix`

Reply:
I named the prefix `HAS_CONTRAST__` since this property is called `hasContrastType` in Neurobagel's [data model](https://github.com/neurobagel/bagel-cli/blob/ecf9ab1c2e176cda3d0bcd0ce140e9093c5561d9/bagel/models.py#L47-L49) :)

I don't mind switching it in the schema to `HAS_SUFFIX__` - what about something like `HAS_IMAGE_SUFFIX__` though for clarity?

Reply:
Renamed to `HAS_IMAGE__` for brevity (also since it applies to a participant-session and not a single acquisition) and updated the column description. Thoughts?